### ISSUE TITLE
Electron Build

### ISSUE THREAD

**Issue by dternyak** [MyEtherWallet/MyEtherWallet#749]
- [x] Create electron build configuration
- [ ] Develop release management strategy for Electron based build, including notifications for updates.
- [x] Address RCEs (detailed internally)
- [x] Disable Ledger support while U2F integration is worked out.
- [x] Ensure private keys / keystore / mnemonic is not disabled on electron

**Comment by jidanni** [comment]
I'll help you test it, if you add a working `--testnet` option...

**Comment by wbobeirne** [comment]
You can always run on the testnet if you select one of the testnet nodes in the top right of the site. That won't be changing with electron.

**Comment by jidanni** [comment]
@wbobeirne It is not obvious to the user which one is Testnet,
```
ETH (MyEtherWallet)
ETH (Etherscan.io)
ETH (infura.io)
ETH (Giveth.io)
ETC (Epool.io)
Ropsten (MyEtherWallet)
Ropsten (infura.io)
Kovan (Etherscan.io)
Kovan (infura.io)
Rinkeby (Etherscan.io)
Rinkeby (infura.io)
EXP (Expanse.tech)
UBQ (ubiqscan.io)
POA (core.poa.network)
TOMO (core.tomocoin.io)
ELLA (ellaism.org)
Add Custom Network / Node
```
I wish somebody would write an article in https://github.com/MyEtherWallet/knowledge-base on how to use MyEtherWallet with Testnet, so we wouldn't have to practice with real ETH.

**Comment by wbobeirne** [comment]
That's a great point, we should make it clear that Ropsten, Kovan, and Rinkeby are Ethereum testnets in the dropdown. And I've just made [an issue about adding that to the knowledge base](https://github.com/MyEtherWallet/knowledge-base/issues/52).

Just a heads up, I'm going to delete this comment and the last 3 in a few hours to keep the issue thread claer and focused on Electron. Wanted to put that out there so it's not confusing if you come back and don't see your comment!

**Comment by wbobeirne** [comment]
Looks like we're all good on this ticket, I'm going to let that second checkbox be addressed in #950.
